### ISSUE TITLE
db upgrade failing: Simulation library file "analysisData-file.phase_scan_test.csv" does not exist

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#2155]
```txt
Jan 08 19:57:12 fnl6b.bivio.biz sirepo_job_supervisor[36579]: File "/home/vagrant/.pyenv/versions/2.7.16/envs/py2/lib/python2.7/site-packages/sirepo/sim_data/__init__.py", line 162, in compute_job_hash
Jan 08 19:57:12 fnl6b.bivio.biz sirepo_job_supervisor[36579]: cls.lib_file_basenames(data))
Jan 08 19:57:12 fnl6b.bivio.biz sirepo_job_supervisor[36579]: File "/home/vagrant/.pyenv/versions/2.7.16/envs/py2/lib/python2.7/site-packages/sirepo/sim_data/__init__.py", line 161, in <genexpr>
Jan 08 19:57:12 fnl6b.bivio.biz sirepo_job_supervisor[36579]: (str(cls.lib_file_abspath(b, data=data).mtime()) for b in sorted(
Jan 08 19:57:12 fnl6b.bivio.biz sirepo_job_supervisor[36579]: File "/home/vagrant/.pyenv/versions/2.7.16/envs/py2/lib/python2.7/site-packages/sirepo/sim_data/__init__.py", line 262, in lib_file_abspath
Jan 08 19:57:12 fnl6b.bivio.biz sirepo_job_supervisor[36579]: sirepo.auth.logged_in_user(),
Jan 08 19:57:12 fnl6b.bivio.biz sirepo_job_supervisor[36579]: sirepo.util.UserAlert: UserAlert(error=Simulation library file "analysisData-file.phase_scan_test.csv" does not exist)
Jan 08 19:57:13 fnl6b.bivio.biz sirepo_job_supervisor[36579]: Traceback (most recent call last):
```

**Comment by robnagler** [comment]
The user has symlinks to a non-existent file

**Comment by robnagler** [comment]
fix 7e50a28daebef1e55dd9b3237b3f300e58c4312b
